modules/test/playwright/tests/object-web/objectEntries.spec.ts

```diff
@@ -45,6 +47,16 @@ export const test = mergeTests(
 	workflowPagesTest
 );
 
+let siteLanguage = 'en';
+
+test.afterEach(async ({accountSettingsPage, page}) => {
+	if (siteLanguage !== 'en') {
+		await page.goto('/en');
```

Comment:
from what i remember, using the `/en` with the slash sometimes caused problems, so could you just use the `en ` without the slash instead? like this:

```
if (siteLanguage !== 'en') {
	await page.goto('en');

	siteLanguage = 'en';
}
```

Reply:
it does, indeed! thanks for noticing!!